examples/nlp/duplex_text_normalization/conf/duplex_tn_config.yaml

```diff
@@ -111,41 +113,36 @@ decoder_exp_manager:
 
 # Data
 data:
-  base_dir: ??? # /path/to/data
-  use_cache: false # uses a cache to store the processed dataset, you may use it for large datasets for speed up (especially when using multi GPUs)
+  base_dir: ??? # /path/to/data # for tagger model
+  use_cache: false # uses a cache to store the processed dataset, you may use it for large datasets for speed up (especially when using multi GPUs) # for tagger model
 
   train_ds:
-    lang: ${lang}
-    data_path: ${data.base_dir}/train.tsv
+    data_path: train.tsv # provide the full path to the file
     batch_size: 64
     shuffle: true
     do_basic_tokenize: false
     max_decoder_len: 80
-    mode: ${mode}
```

Comment:
this will fail when training tagger since it relies on this arg, could you add simple ci test?

Reply:
fix the error